web.Dockerfile

```
@@ -0,0 +1,85 @@
+# Install base packages
+RUN apt-get update -qq && \
+    apt-get install --no-install-recommends -y curl libjemalloc2 libvips gnupg2 libnss3-tools git netcat-openbsd && \
+    rm -rf /var/lib/apt/lists /var/cache/apt/archives
+
+# Install PostgreSQL dependencies
+ARG PG_MAJOR=14
+RUN curl -sSL https://www.postgresql.org/media/keys/ACCC4CF8.asc | \
+    gpg --dearmor -o /usr/share/keyrings/postgres-archive-keyring.gpg \
+    && echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/postgres-archive-keyring.gpg] https://apt.postgresql.org/pub/repos/apt/" \
+    $DISTRO_NAME-pgdg main $PG_MAJOR | tee /etc/apt/sources.list.d/postgres.list > /dev/null
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
```

Hadolint (2.12.0)

[error] Do not use apt-get dist-upgrade

(DL3005)